src/rebar_pkg_resource.erl

```diff
@@ -216,3 +224,11 @@ version_pad([Major, Minor, Patch]) ->
     {list_to_integer(Major), list_to_integer(Minor), list_to_integer(Patch)};
 version_pad([Major, Minor, Patch | _]) ->
     {list_to_integer(Major), list_to_integer(Minor), list_to_integer(Patch)}.
+
+maybe_store_etag_in_cache(false = _UpdateETag, _Path, _ETag) ->
+    ok;
+maybe_store_etag_in_cache(true = _UpdateETag, Path, ETag) ->
+    store_etag_in_cache(Path, ETag).
+
+store_etag_in_cache(Path, ETag) ->
+    ok = file:write_file(Path, ETag).
```

Comment:
Should we crash if we can't write the etag or is it fine to just keep going without it and just have less effective caching?